clang/test/Driver/ppc32-fcomplex-ppc-gnu-abi.c

```diff
@@ -0,0 +1,9 @@
+// RUN: %clang -c --target=ppc32 -fcomplex-ppc-gnu-abi %s 2>&1
```

Comment:
It'd be nice for this to check `-###` output to ensure that it's passing along the correct `-cc1` option.